src/main/java/org/apache/sling/jcr/repoinit/impl/RepositoryInitializerFactory.java

```diff
@@ -99,43 +107,50 @@ public String toString() {
 
     @Override
     public void processRepository(final SlingRepository repo) throws Exception {
-        if ( (config.references() != null && config.references().length > 0)
-           || (config.scripts() != null && config.scripts().length > 0 )) {
+        // loginAdministrative is ok here, definitely an admin operation
+        final Session s = repo.loginAdministrative(null);
+        try {
+            executeScripts(s, config);
+        } catch (Exception e) {
+            if (continueOnError()) {
+                log.error("Repoinit error, won't stop execution because {} is set to non-true. Without this "
```

Comment:
"set to non-true" takes more mental bandwidth than "false" or "set to false", IMHO ;-)